ui/components/app/modals/modal.js

```diff
@@ -356,6 +359,21 @@ const MODALS = {
     },
   },
 
+  ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  SNAP_INSTALL: {
```

Comment:
For future reference, this entire modal system is deprecated and shouldn't be used for new designs. We'll leave this for now due to our time constraints, but this modal system predates the "React portal" feature, which lets us embed modals directly in the UI next to the code that triggers them, rather than use global state to control them at-a-distance which we had to do previously (because we need them to be rendered at the bottom of the document, so they would be shown above the other content with minimal z-index hacks).

Reply:
Do you have any code snippets on how to build modals in the future?

Reply:
Yep, we have a dedicated component for this: `Popover`. There are a few examples of how this is used throughout the codebase.